README.md

```diff
@@ -5,7 +5,7 @@ Terraform
 - [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
 - Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
 
-![Terraform](https://raw.githubusercontent.com/hashicorp/terraform/master/website/source/assets/images/readme.png)
+![Terraform](https://rawgithub.com/hashicorp/terraform/master/website/source/assets/images/logo-hashicorp.svg)
```

Comment:
We can change it, but I'm not super worried about it for this one time. I think camo (GitHub's service) will cache it anyway if I'm not mistaken? Either way, it's a 4-char fix :)